### ISSUE TITLE
Required files of the library running on the browser

### ISSUE THREAD

**Issue by extend1994** [paulrosen/abcjs#79]
Hi!
I am a member of CDNJS. There is a library request on CDNJS and I am the one who is in charge of this. I suppose the files we need to allow users to use the library on a browser are all files under the `bin` directory. If we need some other files or something like that, please tell me. Thank you!

**Comment by paulrosen** [comment]
Yes, those are all the files. In fact, you only need the ones with the latest version. Right now, that is only the files in that folder with "3.1.1" in the title.

How does it work when I do another update and have files with "3.1.2"?

**Comment by extend1994** [comment]
@paulrosen Thank you for your quick response!
So we don't need any version before v3.1.1?
About the filename issue, we get the files of the lib via a mechanism called "auto-update". And we have a special requirement on the filename. That is 

> Filenames should not include a version number and be lowercase.
This is OK: `useful.min.js`, but this is not: `useful-2.0.1.min.js`.

For details about this, you can take a look at https://github.com/cdnjs/cdnjs/blob/master/.github/CONTRIBUTING.md  

Thanks!

**Comment by paulrosen** [comment]
You don't need a version before 3.1.1. Since the files weren't available on the CDN before, there isn't a reason to support the old versions.

I'm not sure what to do about the rest of this. You could use the 3.1.1 files and rename to remove the "3.1.1". There isn't a package.json file that goes with this, so you'd have to create it.

**Comment by extend1994** [comment]
@paulrosen I will create a package.json for the library.
About the filename, is it possible for you to name the files without involving their version in the future? 
I think users can get files of different version if you have tag/release mechanism?
Or we can only add the library manually, which is less effective for us and your users.

Thank you.

**Comment by extend1994** [comment]
@paulrosen 
Can you please also tell me the library is under MIT or GNU GPL v3?
Because it's MIT in GitHub, but GNU GPL v3 in https://abcjs.net/#can-i.
Thanks again.

**Comment by paulrosen** [comment]
Thanks for catching that! It is MIT. I will change the notice on abcjs.net.
